recipes/deltalake/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "deltalake" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deltalake-{{ version }}.tar.gz
+  sha256: b8793a1c7a1219c8935ed9926a6870c9341597396c0f50bf6ff98c323d06ee0f
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - maturin >=0.14,<0.15
+    - make
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}  # [linux]
+  host:
+    - python
+    - maturin >=0.14,<0.15
+    - pip
+    - xz  # [linux]
+  run:
+    - python
+    - pyarrow >=7
+    - typing-extensions  # [py<38]
+
+test:
+  imports:
+    - deltalake
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/delta-io/delta-rs
+  summary: Native Delta Lake Python binding based on delta-rs with Pandas integration
+  license: Apache-2.0
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - charlesbluca
```

@craustin ,  would you still like to be a maintainer on this one?

( given your prior work in PR https://github.com/conda-forge/staged-recipes/pull/22162 )

```suggestion
    - charlesbluca
    - craustin
```

---

We didn't add you as we didn't hear back (though you would still be welcome :)

Feel free to add yourself in the feedstock if this is of interest. Thanks again for your hard work! 🙏

---

Np, sorry for being unresponsive. Thank you for getting this in!